sdk/chain-abstraction/swaps.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: "Swaps"
+description: "Get the best execution when making token swaps"
+---
+
+## Overview
+
+Rhinestone supports two types of swaps: injected and solver-based.
+
+Injected swaps let you to tap into the onchain liquidity using DEX aggregators like 1inch and 0x. They require some additional setup, but provide good token coverage, great liquidity, and best-in-class price execution.
+
+Solver-based swaps use solver liquidity to make swaps. They are simpler to get started, but are limited in tokens they support and bound by the solver liquidity. They are also generally less competitive.
+
+## Injected Swaps
+
+We will use 1inch API to make a WETH -> cbBTC swap on Base chain using funds on Arbitrum:
+
+First, let's initialize 1inch API client:
+
+```ts
+const walletAddress = rhinestoneAccount.getAddress()
+const wethBase = '0x4200000000000000000000000000000000000006'
+const wethAmount = parseEther("0.1")
+const cbbtcBase = '0xcbb7c0000ab88b473b1f5afd9ef808440eed33bf'
+
+const oneInchApiKey = 'YOUR_1INCH_API_KEY'
+
+const config = {
+  tokenAddress: wethBase,
+  amountToSwap: wethAmount,
+  dstToken: cbbtcBase,
+  slippage: 1,
+};
```

Comment:
We don't have the injected swaps supported natively in SDK for now, so going through the manual swap with 1inch here.